opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -161,6 +161,35 @@ mod tests {
     // "multi_thread" tokio flavor must be used else flush won't
     // be able to make progress!
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    async fn counter_overflow_delta() {
+        // Arrange
+        let mut test_context = TestContext::new(Temporality::Delta);
+        let counter = test_context.u64_counter("test", "my_counter", None);
+
+        // Act
+        // Record measurements with A:0, A:1,.......A:1999, which just fits in the 2000 limit
+        for v in 0..2000 {
```

Comment:
It would be good to also test the overflow scenario when a measurement with zero tags has also been recorded.

Reply:
yes, good point. (we'd be expecting 2000+1+1 = 2002 points). Will add next.